pygments/lexers/fortran.py

```diff
@@ -48,19 +48,23 @@ class FortranLexer(RegexLexer):
         ],
         'core': [
             # Statements
+
+            (r'\b(DO)(\s+)(CONCURRENT)\b', bygroups(Keyword, Whitespace, Keyword)),
+            (r'\b(GO)(\s*)(TO)\b', bygroups(Keyword, Whitespace, Keyword)),
```

Comment:
Other whitespace is not given `Whitespace`, so please stay with `Text`.

Reply:
You mean in this particular lexer? I've changed a few to use Whitespace for all whitespace. I presume if I normalize this lexer to use Text.Whitespace you have no objections?

Reply:
Yep!

Reply:
Hi. Do you mean changing:

```
(r'\b(DO)(\s+)(CONCURRENT)\b', bygroups(Keyword, Whitespace, Keyword)),
(r'\b(GO)(\s*)(TO)\b', bygroups(Keyword, Whitespace, Keyword)),
```
to:

```
(r'\b(DO)(\s+)(CONCURRENT)\b', bygroups(Keyword, Text.Whitespace, Keyword)),
(r'\b(GO)(\s*)(TO)\b', bygroups(Keyword, Text.Whitespace, Keyword)),
```

?

Reply:
@Anteru @birkenfeld I added that Text.Whitespace, but I am not sure if this is the correct way to go. If it is not, feel free to close this pull request.